lib/animas/animasSimulator.js

```diff
@@ -202,13 +202,14 @@ exports.make = function(config){
     smbg: function(event) {
 
       if(currSMBG != null && (event.subType === 'manual') && (currSMBG.subType === 'linked')) {
+        var linkedBG = currSMBG.value;
         if(event.units === 'mmol/L') {
           // linked value is always in mg/dL, so we have to convert to
           // mmol/L with one significant digit if we want to compare to
           // Animas mmol/L manual BG value
-          currSMBG.value = (currSMBG.value/18.01559).toFixedNumber(1);
+          linkedBG = (currSMBG.value/18.01559).toFixedNumber(1);
```

Comment:
Could you factor out the conversion as a constant? In viz we've even tried to make sure we only define it once in the repo. Is it being used much or anywhere else currently? (My guess is maybe not as the conversion in most cases is happening back-end...)